src/openloco/platform/platform.windows.cpp

```diff
@@ -45,6 +45,9 @@ namespace openloco::platform
             }
             pszPath.resize(pszPath.size() * 2);
         }
+        auto nullBytePos = pszPath.find(L'\0');
+        if (nullBytePos != std::string::npos)
+            pszPath.resize(nullBytePos);
```

Comment:
I'd prefer to have linebreak before and after. Also, since we're using `wstring`, let's use `wstring:npos`:

```suggestion

        auto nullBytePos = pszPath.find(L'\0');
        if (nullBytePos != std::wstring::npos)
            pszPath.resize(nullBytePos);

```